py/get_tiles.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python
```

Comment:
just the `tilelist` function is needed i guess right? meh whatever i guess it doesnt hurt to have the other stuff your choice :smiley: